CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.27.0 -
+
+-   iOS minimum version updated to 13.0
+-   Disabled mesh visualization and PiP
+-   Fixed a bug where iOS Expert kept ringing after customer had declined to join
```

Comment:
Worth mentioning the change from iOS 11 to iOS 13?

Reply:
> iOS minimum version updated to 13.0
does this not suffice? This is the verbiage provided by Product.

Reply:
D'oh! Somehow my eyes skipped right over that.